cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -28,15 +28,15 @@ rules:
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  namespace: kube-system
+  namespace: logging
```

Comment:
ClusterRoleBinding is not namespaced.
This line should be removed.

Reply:
good catch, thank you, please remove this line @saravanan30erd

Reply:
@coffeepac Its done